source/includes/api/_fields.md

```diff
@@ -48,51 +44,412 @@ curl -X GET https://api.resourcewatch.org/v1/fields/1170030b-9725-4bfe-8fb4-1b0e
+            "dims": 2,
+            "srid": 4326
+        },
+        "the_geom_webmercator": {
+            "type": "geometry",
+            "wkbtype": "Unknown",
+            "dims": 2,
+            "srid": 3857
+        },
+        "objectid": {
+            "type": "number",
+            "pgtype": "int8"
+        },
+        "iso": {
+            "type": "string",
+            "pgtype": "text"
+        },
+        "centroid": {
+            "type": "string",
+            "pgtype": "text"
+        },
+        "area_ha": {
+            "type": "number",
+            "pgtype": "float8"
+        },
+        "topojson": {
+            "type": "string",
+            "pgtype": "text"
+        },
+        "the_geom_simple": {
+            "type": "geometry",
+            "wkbtype": "Unknown",
+            "dims": 2,
+            "srid": 4326
+        },
+        "geojson": {
+            "type": "string",
+            "pgtype": "text"
+        }
+    }
+}
+```
+
+Carto datasets provide a `type` for each field, as well as additional details, depending on the field type. Most fields will have a `pgtype` field, which will identify a sub-type of sorts - for example, `type` number is matched with different `pgtype` values in this example. This is related with the underling Carto implementation, that uses PostgreSQL, PostGIS and its own software stack. We recommend relying on the `type` as much as possible.
+
+Other, more complex, field types, like `geometry` types, include additional details for their respective types. As before, this is related with the underlying Carto implementation tools, and not actively maintained or supported by the RW API.
+
+### ArcGIS feature layer
+
+> Example response for a ArcGIS dataset
+
+```json
+{
+    "tableName": "conservationMapServer3",
+    "fields": {
+        "objectid": {
+            "type": "esriFieldTypeOID"
+        },
+        "tcl_name": {
+            "type": "esriFieldTypeString"
+        },
+        "tcl_id": {
+            "type": "esriFieldTypeSmallInteger"
+        },
+        "area_ha": {
+            "type": "esriFieldTypeInteger"
+        },
+        "tx2_tcl": {
+            "type": "esriFieldTypeSmallInteger"
+        },
+        "gfwid": {
+            "type": "esriFieldTypeString"
+        },
+        "globalid": {
+            "type": "esriFieldTypeGlobalID"
+        },
+        "shape": {
+            "type": "esriFieldTypeGeometry"
+        },
+        "shape_Length": {
+            "type": "esriFieldTypeDouble"
+        },
+        "shape_Area": {
+            "type": "esriFieldTypeDouble"
+        }
+    }
+}
 ```
 
-> Response
+ArcGIS uses its own type naming convention, as you see reflected in the included example. The RW API passes that information as-is to you. As different ArcGIS datasets rely on different ArcGIS server instances and versions, you may encounter variations in the types you'll find, depending on how ArcGIS evolves these types over type.
```

Comment:
Same here, do we have a reference for the possible data types? I found this link, but I'm not sure it is the correct one: https://desktop.arcgis.com/en/arcmap/latest/manage-data/geodatabases/arcgis-field-data-types.htm

Reply:
yes that is the right link @ikas